services/movie.py

```diff
@@ -28,13 +33,14 @@ def create_movie(
     genres_ids: list = None,
     actors_ids: list = None,
```

Comment:
The type hints for `genres_ids` and `actors_ids` are missing. Consider adding `list[int]` or `List[int]` from the `typing` module to maintain consistency and clarity in your function signatures.